docs/DevGuide/BuildSystem.md

```diff
@@ -254,6 +254,15 @@ cmake -D FLAG1=OPT1 ... -D FLAGN=OPTN <SPECTRE_ROOT>
   - Multiply the timeout for the respective set of tests by this factor (default
     is `1`).
   - This is useful to run tests on slower machines.
+- SPECTRE_USE_ALWAYS_INLINE
+  - Force SpECTRE inlining (default is `ON`)
+  - If disabled or if the platform is not using a GNU compiler, replaces
```

Comment:
I wouldn't worry about specifying the GNU part... Clang pretends to be GCC because it's almost completely compatible.. This is basically to guard against Intel and MSVC that have different attributes. I think the replacement is also irrelevant to the user. This is a code-compatibility change. I would just remove this bullet completely. The info in it is way too technical and irrelevant to I think everyone who isn't actively editing the file, in which case they see the code :)